### PR TITLE
en,zh: add backup examples to backup using br docs

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

Original PR: https://github.com/pingcap/docs-tidb-operator/pull/1622
closing #1622 because CLA didn't work. 

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.3 (TiDB Operator 1.3 versions)
- [x] v1.2 (TiDB Operator 1.2 versions)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/backup-to-aws-s3-using-br.md [hunk 1]
```diff
@@ -222,6 +222,182 @@ After you create the `Backup` CR, TiDB Operator starts the backup automatically.
...
+    sendCredToTikv: false
+    clusterNamespace: test1
+    # logLevel: info
+    # statusAddr: ${status_addr}
+    # concurrency: 4
+    # rateLimit: 0
+    # timeAgo: ${time}
+    # checksum: true
+    # options:
+    # - --lastbackupts=420134118382108673
```
Comment (csuzhangxc) [inline en/backup-to-aws-s3-using-br.md:295]: How about removing these lines?

File: en/backup-to-aws-s3-using-br.md [hunk 2]
```diff
@@ -222,6 +222,182 @@ After you create the `Backup` CR, TiDB Operator starts the backup automatically.
...
+  # Only needed for TiDB Operator < v1.1.10 or TiDB < v4.0.8
+  from:
+    host: ${tidb_host}
+    port: ${tidb_port}
+    user: ${tidb_user}
+    secretName: backup-demo1-tidb-secret
```
Comment (csuzhangxc) [inline en/backup-to-aws-s3-using-br.md:301]: How about commenting out these lines? as `# Only needed for TiDB Operator < v1.1.10 or TiDB < v4.0.8`

**Review by csuzhangxc — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**